pkgs/games/factorio/utils.nix

```diff
@@ -18,7 +18,9 @@ with lib;
           # NB: there will only ever be a single zip file in each mod derivation's output dir
           ln -s $modDrv/*.zip $out
         done
-      '';
+      '' + (if modsDatFile != null then ''
```

Comment:
Possibly lib.optionalString would make this slightly nicer but on the whole I don't mind either way.